examples/generate_ast.py

```diff
@@ -0,0 +1,44 @@
+# -----------------------------------------------------------------
+# pycparser: generate_ast.py
+#
+# Tiny example of writing an AST from scratch to C code.
+#
+# Andre Ribeiro [https://github.com/Andree37]
+# License: BSD
+# -----------------------------------------------------------------
+from __future__ import print_function
+
+from pycparser import c_ast, c_generator
+
+
+# target C code:
+# int main() {
+#     return 0;
+# }
+
+
+def create_your_ast():
+    constant_zero = c_ast.Constant(type='int', value='0')
+    return_node = c_ast.Return(expr=constant_zero)
+    compound_node = c_ast.Compound(block_items=[return_node])
+    type_decl_node = c_ast.TypeDecl(declname='main', quals=[], type=c_ast.IdentifierType(names=['int']), align=[])
```

Comment:
pycparser code typically tries to wrap at 80 columns
should be easy here with all the keyword params (like you do in func_def_node)